app/modules/ads/index.js

```diff
@@ -179,20 +142,14 @@ class Ads {
   finishFirstCall() {
     const { btfBlockerService } = window.Wikia.adEngine;
 
-    if (this.isAdStackEnabled()) {
-      btfBlockerService.finishFirstCall();
-      fanTakeoverResolver.resolve();
-    }
+    btfBlockerService.finishFirstCall();
+    fanTakeoverResolver.resolve();
   }
 
   createJWPlayerVideoAds(options) {
     const { jwplayerAdsFactory } = window.Wikia.adProducts;
 
-    if (this.isAdStackEnabled()) {
-      return jwplayerAdsFactory.create(options);
-    }
-
-    return null;
+    return jwplayerAdsFactory.create(options);
```

Comment:
What if there is no ad stack on FV page?

Reply:
Never mind, I've noticed your changes: 
https://github.com/Wikia/mobile-wiki/pull/1821/files#diff-1f8a4e7742e20ee4da173fdc8f902a2fR71